frontend/src/App.jsx

```diff
@@ -39,6 +40,7 @@ function App() {
               path="/profilethree"
               Component={withAuth(ProfileCompletionThree)}
             />
+            <Route path="/dashboard" element={<Dashboard />} />
```

Comment:
wrap in withAuth  function since we don't want people who haven't logged in to be enter this